addons/mail/tests/common.py

```diff
@@ -581,8 +581,20 @@ def assertSentEmail(self, author, recipients, **values):
                     email_to_list.append(formataddr((email_to.name, email_to.email or 'False')))
                 else:
                     email_to_list.append(email_to)
-        expected['email_to'] = email_to_list
+        formatted_emails = []
+        for email_to in email_to_list:
+            emails_split = email_to.split(', ')
+            reconstructed = []
+            for email_split in emails_split:
+                name_mail = email_split_tuples(email_split)
+                if name_mail == []:
+                    reconstructed.append(email_split)
+                else:
+                    name, mail = name_mail[0]
+                    reconstructed.append(formataddr((name.replace('@', '').strip("' "), mail)))
```

Comment:
if we change `formataddr`, we might be able to use `email_split_and_format` here